src/go/rpk/pkg/config/config_test.go

```diff
@@ -275,6 +275,90 @@ func TestWriteConfig(t *testing.T) {
 		wantErr      bool
 		expected     string
 	}{
+		{
+			name: "shall write a valid config file without advertised_kafka_api",
```

Comment:
shouldn't there be one extra test that gets the default configuration and it should have nil advertised ips?

Reply:
Solved!